examples/llm/openai_component_request_response.yaml

```diff
@@ -0,0 +1,146 @@
+# This example demostrates how to use the request_response_flow_controller to 
+# inject another flow into an existing flow. This is commonly used when
+# you want to call a service that is only accessible via the broker.
+#
+#    Main flow:   STDIN -> llm_streaming_custom_component -> STDOUT
+#                                          |
+#                                          v
+#                              do_broker_request_response()
```

Comment:
Doesn't this wait for the response before going to STDOUT? The graphs convey it branches into 2 different flows. Maybe add a return arrow from `do_broker_request_response`

Reply:
Good point. fixed